CoffeeChat/SceneDelegate.swift

```diff
@@ -6,18 +6,31 @@
 //  Copyright © 2020 cuappdev. All rights reserved.
 //
 
+import GoogleSignIn
 import UIKit
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
 
-
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
```

Comment:
I'm under the impression that SceneDelegate is a new iOS 13 thing for managing different windows of the app (like on the iPad or something). Don't we want to move the root view controller stuff into AppDelegate under didfinishlaunch?

Reply:
I believe using `SceneDelegate` is correct
https://stackoverflow.com/questions/56498099/difference-between-scenedelegate-and-appdelegate